projects/sdk-core/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.log
+.DS_Store
+node_modules
+dist
+cache
+.env
```

Comment:
We need to ignore `src/constants/generated/**/*` and `src/constants/generated-gql/**/*` also don't we? The `generate` command in sdk-core would leave artifacts there.